scripts/content.js

```diff
@@ -0,0 +1,25 @@
+let posts = document.querySelectorAll('.update-components-text')
+let main = document.querySelector('.scaffold-finite-scroll__content')
+let config = {childList: true, characterData: true, subtree: true}
+
+const checkForRestrictedWords = (text, post) => {
+    if (text.innerText.includes(" AI ")) {
+        console.log('post includes restricted word')
+        console.log(text.innerText)
+        post.classList.add('hide')
+    }
+}
+
+const callback = (mutationList, observer) => {
+    mutationList.forEach(mutation => {
```

Comment:
This code iterates through every mutation and checks to see if the target contains a specific class. If it does, it then invokes `checkForRestrictedWords`.